psydac/linalg/tests/test_linalg.py

```diff
@@ -522,7 +530,7 @@ def test_in_place_operations(n1, n2, p1, p2, P1=False, P2=False):
     assert isinstance(Z3, StencilMatrix)
     assert isinstance(T, StencilMatrix)
     assert np.array_equal(w2.toarray(), np.dot(np.dot(2, Sa), v_array))
-    
+test_in_place_operations(10, 10, 2, 2, P1=False, P2=False)   
```

Comment:
If you really want this line to be run interactively with
```shell
$ python3 test_linalg.py
```
then you should move it to the bottom of the file:
```python
if __name__ == "__main__":
    ...
    test_in_place_operations(10, 10, 2, 2, P1=False, P2=False)
```
I am not sure that this is useful though.

Reply:
Sorry I must have commented on an old commit.